src/main/java/net/imagini/zkcli/ZkCli.java

```diff
@@ -51,6 +51,7 @@ private void doCliActions() {
                                                       connectionMethodInvalidException);
                 }
             }
+            // TODO: don't assume a cluster is always specified
```

Comment:
Also as for the undefined behaviour with the zookeeper node. Curator will not let you delete the root "zookeeper" znode so I think we should probably have an explicit filter to just not try deleting that one if deleting children so that people can clear their Zookeeper. Maybe with a stderr message "Skipping protected node - zookeeper" though that's not really necessary. Most if not all zk tools have that limitation so people don't expect the zookeeper node to be deleted. Similarly deleting or recursively deleting / should not be permitted but in those cases we should exit 1 (i.e. throw an exception).